src/step.rs

```diff
@@ -272,6 +281,28 @@ pub mod step_routes {
             )
         })
     }
+
+    #[get("/main")]
+    fn continue_to_main(
+        sender: rocket::State<'_, crate::PrirodaSender>,
+    ) -> crate::RResult<response::Flash<response::Redirect>> {
+        sender.do_work(move |pcx| {
+            let main_id = pcx
+                .ecx
+                .tcx
+                .tcx
+                .entry_fn(LOCAL_CRATE)
+                .expect("no main or start function found")
+                .0
+                .to_def_id();
+            pcx.config
+                .bptree
+                .add_breakpoint(Breakpoint(main_id, mir::BasicBlock::new(0), 0));
```

Comment:
Can you replace `ShouldContinue::Continue` with a check if it is at the start of main below instead?

Reply:
Yeah, I thought the impl was going to be more complicated which is why I took this approach. I can do that